pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -61,6 +61,16 @@ func (r *ReconcileStorageCluster) ensureStorageClasses(instance *ocsv1.StorageCl
 	return nil
 }
 
+// ensureExternalClusterConfig ensures that Storagclass for the external cluster exist
+func (r *ReconcileStorageCluster) ensureExternalClusterConfig(instance *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	// blob := `[{"kind":"ConfigMap","data":{"maxMonId":"0","data":"a=10.106.31.93:6789","mapping":{}},"name":"rook-ceph-mon-endpoints"},{"kind":"Secret","data":{"mon-secret":"mon-secret","fsid":"ac1235ba-c390-48d7-8e1f-f2ece695b15b","cluster-name":"openshift-storage","admin-secret":"admin-secret"},"name":"rook-ceph-mon"},{"kind":"Secret","data":{"userKey":"AQBB8rNekvsHGhAA8sqfEaDE07JLABxOJ87\/Kg==","userID":"client.healthchecker"},"name":"rook-ceph-operator-creds"},{"kind":"Secret","data":{"userKey":"AQBV66pegXNrARAAoHTLdkHamWESQLEnNUzwaw==","userID":"csi-rbd-node"},"name":"rook-csi-rbd-node"},{"kind":"Secret","data":{"userKey":"AQBU66peVcBCKxAAoPQ4tKlJ9gxvrH3MRnDRHQ==","userID":"csi-rbd-provisioner"},"name":"rook-csi-rbd-provisioner","pool-name":"leseb"},{"kind":"Secret","data":{"userKey":"AQBV66pedfkUIhAAn\/tnB0cvIih5n9DwwxOwBg==","userID":"csi-cephfs-node"},"name":"rook-csi-cephfs-node"},{"filesystem-name":"myfs","kind":"Secret","data":{"userKey":"AQBV66pefnqmERAAKYhoO2XK5mUIGKSN4J\/URw==","userID":"csi-cephfs-provisioner"},"name":"rook-csi-cephfs-provisioner","pool-name":"myfs-data0"},{"kind":"StorageClass","data":{"endpoint":"192.168.0.1:80"},"name":"rook-ceph-retain-bucket"}]`
+	_, err := r.newStorageClasses(instance)
```

Comment:
@umangachapagain can you please let me know where and how to create resources from this test resources meanwhile since the script PR is in WIP?. Thanks